aspnetcore/data/ef-mvc/intro.md

```diff
@@ -124,6 +130,8 @@ In the following sections you'll create a class for each one of these entities.
 
 ![Student entity diagram](intro/_static/student-entity.png)
 
+In the project folder, create a *Models* folder.
```

Comment:
My mistake - on my first pass I was planning to rewrite the tutorial to select no auth, then I switched back to auth when I realized how much work the change was going to require, but missed removing this comment.